retirement_api/utils/ss_calculator.py

```diff
@@ -72,6 +73,12 @@ def get_note(note_type, language):
 
 comment = re.compile(r"<!--[\s\S]*?-->")  # regex for parsing indexing data
 
+# calculation constants
+EARLY_PENALTY = 0.00555555  # monthly penalty for months closest to FRA
+EARLIER_PENALTY = 0.004166666  # monthly penalty for earliest claiming ages
+MONTHLY_BONUS = 0.00667  # bonus value for each month's delay past FRA
+ANNUAL_BONUS = 0.08  # annual bonus value for each year's delay past FRA
+
```

Comment:
This is just me being curious, no changes needed: are all the magical numbers used in this app published somewhere by SSA or something? How do you figure this out?